desc.en.hocon

```diff
@@ -4353,7 +4353,7 @@
       "label": "RPC Socket Sndbuf"
     },
     "rpc_ssl_server_port": {
-      "desc": "Listening port used by RPC local service.<br/>\nNote that this config only takes effect when rpc.port_discovery is set to manual\nand <code>driver</code> is set to <code>ssl</code>.",
+      "desc": "Listening port used by RPC local service.<br/>Note that this config only takes effect when `rpc.port_discovery` is set to manual and `rpc.driver` is set to `ssl`.",
```

Comment:
this has been changed in upstream: https://github.com/emqx/emqx/pull/13434
please consider sending a new commit to my PR instead.